corehq/apps/ota/utils.py

```diff
@@ -219,3 +224,26 @@ def _inner(request, domain, *args, **kwargs):
 
         return response
     return _inner
+
+
+def can_login_on_device(user_id, device_id):
+    if device_id.startswith("WebAppsLogin"):
+        return True
+
+    end_time = datetime.now()
```

Comment:
Should this use `utcnow()` or whatever the acceptable equivalent is these days?

Aside: I find it annoying that `utcnow()` is deprecated. It's so easy to remember.

Reply:
Ah right. I believe it is now datetime.now(timezone.utc) based on [these docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow). See f90aa8146a881e7090628dad6ab0e660f47b9e22

Reply:
In Django world you can also use `django.utils.timezone.now()`